### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jabrena** [gungwald/boinc-jni#1]
Hi, 

do you have some documentation about your project?

**Comment by gungwald** [comment]
I do not. It's not functional yet. But, it is close. I could finish it up if you're interested in using it.
 

    On Saturday, July 15, 2017 7:11 PM, Juan Antonio Breña Moral <notifications@github.com> wrote:
 

 Hi,do you have some documentation about your project?—

**Comment by jabrena** [comment]
Yes, I would like to use boinc from java to participate in `SETI@Home`.

**Comment by gungwald** [comment]
I'm afraid boinc-jni won't help with that. If you want to participate in SETI@Home you should follow the instructions at https://setiathome.berkeley.edu/join.php. They have clients for Windows, Mac, and Linux. What boinc-jni does is allow you to create new projects like SETI@Home, but with Java instead of C++.
